live_flights_on_map.md

```diff
@@ -0,0 +1,125 @@
+## Documentation for live flight details on a map
+
+##### 1. Map Creation
+
+Firstly, we have to insert OpenLayers css into an HTML file:
+
+```html
+<link rel="stylesheet"href="https://cdn.jsdelivr.net/gh/openlayers/openlayers.github.io@master/en/v6.1.1/css/ol.css"type="text/css">
+```
+
+Then we have to insert OpenLayers js file:
+
+```html
+<script src="https://cdn.jsdelivr.net/gh/openlayers/openlayers.github.io@master/en/v6.1.1/build/ol.js"></script>
+```
+
+Inside `body` of the html file, we have to create a `map` \<div>:
+
+```html
+<div id="map" class="map"></div>
+``` 
+
+In the above `map` \<div>, our map will reside.
+
+Next, we will add an OSM map into the map:
+
+```javascript
+    <script type="text/javascript">
+      var map = new ol.Map({
+        target: 'map',
+        layers: [
+          new ol.layer.Tile({
+            source: new ol.source.OSM()
+          })
+        ],
+        view: new ol.View({
+          center: ol.proj.fromLonLat([37.41, 8.82]),
+          zoom: 4
+        })
+      });
+    </script>
+```
+
+
+Final HTML will look like this:
+```html
+<!doctype html>
+<html lang="en">
+  <head>
+    <link rel="stylesheet"href="https://cdn.jsdelivr.net/gh/openlayers/openlayers.github.io@master/en/v6.1.1/css/ol.css"type="text/css">
+    <style>
+      .map {
+        height: 400px;
+        width: 100%;
+      }
+    </style>
+    <script src="https://cdn.jsdelivr.net/gh/openlayers/openlayers.github.io@master/en/v6.1.1/build/ol.js"></script>
+    <title>OpenLayers example</title>
+  </head>
+  <body>
+    <h2>My Map</h2>
+    <div id="map" class="map"></div>
+    <script type="text/javascript">
+      var map = new ol.Map({
+        target: 'map',
+        layers: [
+          new ol.layer.Tile({
+            source: new ol.source.OSM()
+          })
+        ],
+        view: new ol.View({
+          center: ol.proj.fromLonLat([37.41, 8.82]),
+          zoom: 4
+        })
+      });
+    </script>
+  </body>
+</html>
+
+```
+
+##### 2. Placing the flight details
+
```

Comment:
@arnabdutta73 can you give a bit more details about the API you are using here and also a link to the ajax documentation and the use of it here.

Reply:
:+1: